pos_ticket_partner_firstname/static/src/xml/pos.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<templates id="template" xml:space="preserve">
+
+    <t t-extend="PosTicket">
```

Comment:
until Odoo 13 (I think) there are two ticket format. ``PosTicket`` and the ``XmlReceipt``. 
could you handle both cases in your module? 

https://github.com/odoo/odoo/blob/12.0/addons/point_of_sale/static/src/xml/pos.xml#L736

if it's not in the scope of your developpement, please add a know issues in the roadmap file to mention that this module does'nt work in a non proxy mode.

thanks !

Reply:
It's already handled, by overloading `export_for_printing` on the js file :)

Reply:
thanks !